src/fix/main.c

```diff
@@ -652,6 +652,20 @@ static ssize_t writeBytes(int fd, void *buf, size_t len)
 
 	return total;
 }
+/**
+ * @param rom0 A pointer to rom0
+ * @param startAddr What address to begin checking from
+ * @param size How many bytes to check
+ * @param areaName Name to be displayed in the warning message
+ */
+static void nonZeroWarn(uint8_t *rom0, uint16_t startAddr, uint8_t size, char const *areaName) {
+	for (uint8_t i = 0; i < size; i++) {
+		if (rom0[i + startAddr] != 0) {
+            fprintf(stderr, "warning: Overwrote one or more non-zero bytes in the %s\n", areaName);
```

Comment:
Fix the indentation of `fprintf`. Note that `areaName);` will wrap onto its own line to keep the maximum line length at 100 characters assuming 8-wide tabs. (The `areaName);` would be indented with a extra tab, aligning it inside the `fprintf(` parentheses with `stderr`.)

Reply:
Also, I agree with @ISSOtm that you don't need an exact non-zero byte count, but "one or more" sounds odd to me for single-byte areas. How about `size > 1 ? "one or more non-zero bytes" : "the non-zero byte"`?